File: analisis_basico/aleatorio/aleatorio.py
import random

MIN = 0
MAX = 1000000
ITER = 100000000

def generateRandom():
    
    random_num = 0
    for _ in range (ITER):
        # genera un numero aleatorio entre MIN y MAX (ambos incluidos)
        random_num = random.randint(MIN, MAX)


if __name__ == "__main__":
    generateRandom()

File: analisis_basico/aleatorio/aleatorio.java
import java.util.Random;

public class aleatorio {

    private static final int MIN = 0;
    private static final int MAX = 1000000;
    private static final int ITER = 100000000;

    public static void generateRandom() {
        // inicializa el generador de numeros aleatorios
        Random random = new Random();
        
        int random_num;
        for (int i = 0; i < ITER; i++) {
            // genera un numero aleatorio entre MIN y MAX - 1
            random_num = random.nextInt(MAX - MIN + 1) + MIN;
        }
    }

    public static void main(String[] args) {
        generateRandom();
    }
}